src/x86/sse41.rs

```diff
@@ -15,6 +18,174 @@ pub unsafe fn _mm_blendv_epi8(
     pblendvb(a, b, mask)
 }
 
+#[inline(always)]
+#[target_feature = "+sse4.1"]
+#[cfg_attr(test, assert_instr(pblendw, imm8=0xF0))]
+pub unsafe fn _mm_blend_epi16(a: i16x8, b: i16x8, imm8: u8) -> i16x8 {
```

Comment:
Should the `imm8` be `u8` or `i32`? The public interface in C is `const int`, but I followed [`_mm_dp_ps`](https://github.com/p32blo/stdsimd/blob/08574d8f02395f417cab2a179cf8fae326fa1126/src/x86/sse41.rs#L216) for consistency and used `u8`.

Reply:
I'd imagine that so long as more than 8 bits aren't read then passing `u8` here seems fine to me, but @BurntSushi do you think we should super strictly follow C?